draft-opsarea-rfc5706bis.md

```diff
@@ -190,10 +183,14 @@ contributor:
    how threats should be mitigated, and then suggests appropriate standard
    protocols that could mitigate the threats.
 
-   This document does not impose a specific management or operational solution, imply that a formal Data
-   Model is needed, or imply that using a specific management protocol
-   is mandatory. If Protocol Designers conclude that the technology can be managed
-   solely by using Proprietary Interfaces or that it does
+   Separate manageability and operational considerations sections are desirable
```

Comment:
I think we need to delete this sentence is we have all in one single section, with a recommended location in the document. 

Reasoning about separate sections will confuse readers.